docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,13 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* has a need to take notes
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: Take and manage notes faster than a typical mouse/GUI driven app
```

Comment:
Should we add in the directory-like structure and other features as our proposition? Not sure how to phrase them tho